src/server/Doorman.java

```diff
@@ -56,6 +60,9 @@
 	public Doorman(CommandHandler commandHandler, int port) throws IOException {
 
 		this.commandHandler = commandHandler;
+		// TODO: Don't hard-code this location.
```

Comment:
There is actually a constant identifier for the resource path in Executor but it's not widely used yet.